.github/workflows/build-release-image.yaml

```diff
@@ -41,6 +41,11 @@ jobs:
           docker-compose build document-utils-api 
           docker tag jag-document-utils_document-utils-api:latest ${{ secrets.DOCKERHUB_ORG }}/document-utils-api:latest
           docker push ${{ secrets.DOCKERHUB_ORG }}/document-utils-api:latest
+<<<<<<< Updated upstream
```

Comment:
I think you're committing merge conflicts here

Reply:
shot